desc/equilibrium/configuration.py

```diff
@@ -837,12 +886,143 @@ def pressure(self, new):
     @property
     def p_l(self):
         """ndarray: Coefficients of pressure profile."""
-        return self.pressure.params
+        return np.empty(0) if self.pressure is None else self.pressure.params
 
     @p_l.setter
     def p_l(self, p_l):
+        if self.pressure is None:
+            raise ValueError(
+                "Attempt to set pressure on an equilibrium "
+                + "with fixed kinetic profiles"
+            )
         self.pressure.params = p_l
 
+    @property
+    def electron_temperature(self):
+        """Profile: Electron temperature (eV) profile."""
+        return self._electron_temperature
+
+    @electron_temperature.setter
+    def electron_temperature(self, new):
+        if isinstance(new, Profile) or (new is None):
+            self._electron_temperature = new
```

Comment:
Could we use `self.parse_profile` here to allow other input formats besides Profile types? Same for all the other profile setter methods.